yogstation/code/game/objects/items/devices/powersink.dm

```diff
@@ -1,4 +1,7 @@
 /obj/item/powersink/examine(mob/user)
 	. = ..()
 	if(isobserver(user))
-		to_chat(user, "The power dial reads [power_drained] J.")
\ No newline at end of file
+		to_chat(user, "The power dial reads [power_drained] J.")
+
+/obj/item/powersink/multitool_act(mob/user)
+	to_chat(user, "The power dial reads [power_drained] J.")
```

Comment:
if you don't add a Return TRUE it'll also hit the machine, after doing the multitool_act